app/views.py

```diff
@@ -103,13 +113,34 @@ def createquiz():
     form = CreateQuiz()
     return render_template('createquiz.html', title=' | Create Quiz', form=form)
 
+@app.route('/createnewquiz', methods=['POST'])
+def createnewquiz():
+    """Routing for creating a new quiz."""
+    form = CreateQuiz()
+    #if form.validate_on_submit():
+        # Commit inputs to database
+    return render_template('createquiz.html', title=' | Create Quiz', form=form)
```

Comment:
Why does this need to be separate routing from `createquiz`? There is confusion between adding questions and creating quiz, do make it clear in the code.

Reply:
So if I understand this right but correct me:

Create Quiz Form has 2 submit buttons --> One is to create the quiz, and the other is to add a new question. So the add a new question, I am assuming the question will be submitted to the database as well.

The other button will "create the quiz" which just adds up all the qna from add new questions

Reply:
I'm confused. At which point does a quiz exist? If quiz and question do not get created at the same time please use distinct forms.